src/core/configuration.c

```diff
@@ -358,7 +358,7 @@ MsQuicConfigurationLoadCredential(
 _IRQL_requires_max_(PASSIVE_LEVEL)
 void
 QuicConfigurationTraceRundown(
-    _In_ QUIC_CONFIGURATION* Configuration
+    _In_ QUIC_CONFIGURATION* Configuration // NOLINT
```

Comment:
What does the `// NOLINT` do here?

Reply:
It stops a warning that `Configuration` is unused, because on Linux `QuicTraceEvent` is a no-op.

Reply:
I'd prefer `UNREFERENCED_PARAMETER` instead, since that's what we do everywhere else.

Reply:
I can make that change (also elsewhere). But don't you think it's confusing, since the parameter is actually referenced when `QuicTraceEvent` isn't a no-op?